tiptapy/__init__.py

```diff
@@ -75,22 +75,26 @@ def inner_render(self, node) -> str:
 
 class Image(BaseNode):
     type = "image"
-    wrap_tag: str = "figure"
 
     def inner_render(self, node) -> str:
         special_attrs_map = {'caption': 'figcaption'}
         attrs = node.get("attrs", {})
+        html = ""
         attrs_s = " ".join(f'{k}="{v}"'
                            for k, v in attrs.items()
                            if k not in special_attrs_map and v.strip()
                            )
-        html = f"<img {attrs_s}>"
-        if attrs.get('caption', '').strip():
-            tag = special_attrs_map['caption']
-            html += f"<{tag}>{attrs['caption']}</{tag}>"
+        if attrs_s:
+            inner_html = f"<img {attrs_s}>"
+            caption = attrs.get('caption', '').strip()
+            if caption:
```

Comment:
@Pradhvan no need to create extra variable just to check condition.

Reply:
@gauravr updated and removed the extra variable. On a personal preference, here I did find explicitly creating a variable and using it in the condition to be a bit more rediable code.